lib/postgrex/extensions/interval.ex

```diff
@@ -52,7 +52,8 @@ defmodule Postgrex.Extensions.Interval do
     def decode(type) do
       quote location: :keep do
```

Comment:
You can also do:

```suggestion
      quote location: :keep, generated: true do
```

and not have to worry about pleasing Dialyzer. :)